app_dev.yaml

```diff
@@ -247,6 +247,10 @@ libraries:
 
 env_variables:
   PYTHONHTTPSVERIFY: 1
+  GAE_USE_SOCKETS_HTTPLIB : "anyvalue"
+
+vpc_access_connector:
```

Comment:
Maybe add a comment explaining this is for memcache and pointing to the relevant GAE official doc.

Reply:
Done.